data/data-pipeline/data_pipeline/etl/sources/maryland_ejscreen/etl.py

```diff
@@ -0,0 +1,139 @@
+            gpd.read_file(f)
+            for f in list_of_files
+            if not f.endswith("CountiesEJScore.shp")
+        ]
+
+        # Set the Census tract as the index and drop the geometry column
+        # that produces the census tract boundaries.
+        # The latter is because Geopandas raises an exception if there
+        # are duplicate geometry columns.
+        # Moreover, since the unit of measurement is at the tract level
+        # we can consistantly merge this with other datasets
+        dfs_list = [
+            df.set_index("Census_Tra").drop("geometry", axis=1)
+            for df in dfs_list
+        ]
+        # pylint: disable=unsubscriptable-object
+        combined_df = gpd.GeoDataFrame(pd.concat(dfs_list, axis=1))
+
+        # Reset index so that we no longer have the tract as our index
+        combined_df = combined_df.reset_index()
+        # coerce into integer into
+        # pylint: disable=unsupported-assignment-operation, unsubscriptable-object
+        combined_df["Census_Tra"] = (combined_df["Census_Tra"]).astype(int)
+
+        # Drop the 10 census tracts that are zero: please see here:
+        # https://github.com/usds/justice40-tool/issues/239#issuecomment-995821572
+        combined_df = combined_df[combined_df["Census_Tra"] != 0]
+
+        # Set our class instance variable after conversions in lines 50-81
+        self.df = combined_df.copy()
+
+        # Rename columns
+        self.df.rename(
+            columns={
+                "Census_Tra": self.GEOID_TRACT_FIELD_NAME,
+                "EJScore": field_names.MARYLAND_EJSCREEN_SCORE_FIELD,
+            },
+            inplace=True,
+        )
+
+        # Interpretation: An EJ score (reported as a percentile)
+        # has a percentile rank of N for some N between 0 - 100"
+        # Please see the README and the following linksfor details and context
+        # 1. https://github.com/usds/justice40-tool/issues/239#issuecomment-1003448074
+        # 2. https://github.com/usds/justice40-tool/issues/239#issuecomment-995798345
+        self.df[
+            field_names.MARYLAND_EJSCREEN_SCORE_FIELD
+            + field_names.PERCENTILE_FIELD_SUFFIX
+        ] = self.df[field_names.MARYLAND_EJSCREEN_SCORE_FIELD].rank(
+            pct=True, ascending=True
+        )
+
+        # An arbitrarily chosen threshold is used in the comparison tool output
+        self.df[field_names.MARYLAND_EJSCREEN_BURDENED_THRESHOLD_FIELD] = (
+            self.df[
+                field_names.MARYLAND_EJSCREEN_SCORE_FIELD
+                + field_names.PERCENTILE_FIELD_SUFFIX
+            ]
+            > 0.75
+        )
+
+        # Baseline Comparisons with some quartiles and the 90th percent OF EJ Score
+        # Interpretation: The score is greater than or equal to N% of the tracts in the state.
+        # Please see the README for details on the EJScore interpretation
+        self.df[field_names.MARYLAND_EJSCREEN_TRACT_25_PERCENTILE_FIELD] = (
```

Comment:
I'm a little surprised that we're still talking about this, but these are NOT percentiles. MARYLAND_EJSCREEN_SCORE_FIELD is not a percentile so labeling these as percentiles is misleading. 

If you're worried about this code, you can also just drop these four fields and just keep `field_names.MARYLAND_EJSCREEN_BURDENED_THRESHOLD_FIELD`.

Reply:
I will do as you have suggested. I am merely reflecting on what is in the papers and the documentation that was published by the research team.

Reply:
Bump. Doesn't seem like you addressed this comment?

Reply:
I was trying to reconcile this with the earlier comment here: https://github.com/usds/justice40-tool/pull/1093#discussion_r782537285

I have gone ahead and just removed all references in the source code. The reader will just have to review the README and the papers.